### PR TITLE
[Paimon-818][Hive]add ITCase for hive on s3.

### PR DESCRIPTION
**Pull request by hk-lrzy**
*(Please specify the module before the PR name: [core] ... or [flink] ...)*

### Purpose

*(What is the purpose of the change, or the associated issue)*

### Tests

*(List UT and IT cases to verify this change)*

### API and Format 

*(Does this change affect API or storage format)*

### Documentation

*(Does this change introduce a new feature)*

### CODE REVIEW

File: paimon-hive/paimon-hive-connector-common/src/test/java/org/apache/paimon/hive/HiveS3ITCase.java [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+/** ITCase for using S3 in Hive. */
+@RunWith(PaimonEmbeddedHiveRunner.class)
+@ExtendWith(ParameterizedTestExtension.class)
+public class HiveS3ITCase {
```
Comment (JingsongLi) [inline paimon-hive/paimon-hive-connector-common/src/test/java/org/apache/paimon/hive/HiveS3ITCase.java:65]: Can this class extends `HiveCatalogITCaseBase`?

**Comment by JingsongLi** [comment]
Test failed, you can see in: https://github.com/hk-lrzy/incubator-paimon/tree/paimon-818

**Comment by wg1026688210** [comment]
hi~ @hk-lrzy   You can rebase the master branch. I improved the PaimonEmbeddedHiveRunner for sovling a concurrency problem and adding  a MinioContainer.  This be useful for you, and don't forget to add a licensing header at  new file

**Comment by wg1026688210** [comment]
> hi~ @hk-lrzy You can rebase the master branch. I improved the PaimonEmbeddedHiveRunner for sovling a concurrency problem and adding a MinioContainer. This may  be useful for you, and don't forget to add a licensing header at new file

**Comment by JingsongLi** [comment]
It seems this pr is not active, I close this now. Feel free to re-open this if you come back.

**Pull request closed without merging**